mlflow/odahuflow/trainer/helpers/mlflow_helper.py

```diff
@@ -258,37 +258,47 @@ def _remember_cwd():
 
 
 def mlflow_to_gppi_cli():
-    parser = argparse.ArgumentParser()
 
-    def dir_type(string):
-        os.makedirs(string, exist_ok=True)
+    def make_dir(string):
+        try:
+            os.makedirs(string, exist_ok=True)
+        except FileExistsError as error:
+            raise ValueError(f'A file already exists with the same name: {string}\n'
+                             'Rename file or directory and try again.') from error
+
+    def validate_dir(string):
         if not os.path.isdir(string):
             raise ValueError(f'{string} is not a valid directory path')
-        return string
+
+    parser = argparse.ArgumentParser(description='Converts MLFLow model to GPPI.')
 
     parser.add_argument('--verbose', action='store_true', help='More extensive logging')
     parser.add_argument('--model-name', type=str, required=True, help='Name of GPPI Model')
     parser.add_argument('--model-version', type=str, required=True, help='Version of GPPI Model')
-    parser.add_argument('--mlflow-model-path', '--mlflow', required=True, type=dir_type,
-                        help='Path to source MLFlow model directory')
+    parser.add_argument('--mlflow-model-path', '--mlflow', required=True,
+                        type=str, help='Path to source MLFlow model directory')
     parser.add_argument('--gppi-model-path', '--gppi', required=True,
-                        type=dir_type, help='Path to result GPPI model directory')
-    parser.add_argument('--tgz', required=False, default=True,
-                        type=bool, help='Make tar arhieve with gppi folder')
-
+                        type=str, help='Path to result GPPI model directory')
+    parser.add_argument('--tgz', dest='tgz', action='store_true', help='Make tar arhieve with gppi folder')
+    parser.add_argument('--no-tgz', dest='tgz', action='store_false')
+    parser.set_defaults(tgz=True)
```

Comment:
```suggestion
    parser.add_argument('--no-tgz', dest='tgz', action='store_false', help='Prevent archiving result directory')
```
Using `action='store_false'` is indeed appropriate solution, but the rest is a bit overloaded in my opinion. I would suggest just to leave `--no-tgz` option. Introducing both `--tgz` and `--no-tgz` is an overhead, from my point of view.
Setting defaults is also unnecessary:
```
>>> parser = argparse.ArgumentParser()
>>> parser.add_argument('--no-bar', action='store_false', dest='bar')
>>> parser.parse_args('--no-bar'.split())
Namespace(bar=False)
>>> parser.parse_args(''.split())
Namespace(bar=True)
```

Reply:
Agree. Commited